app/scripts/dimApp.i18n.js

```diff
@@ -99,19 +99,19 @@
           General: "Allgemein",
           Postmaster: "Poststelle",
           Equip: "Ausrüsten",
-          Store: "Speichern",
+          Store: "In den Tresor legen",
           Vault: "Tresor",
           Vanguard: "Vorhut",
           Reputation: "Ruf",
           Intellect: "Intellekt",
           Discipline: "Disziplin",
           Strength: "Stärke",
-          Super_cooldown: "Super abklingzeit",
-          Grenade_cooldown: "Granaten abklingzeit",
-          Melee_cooldown: "Nahkampf abklingzeit",
+          Super_cooldown: "Super Abklingzeit",
+          Grenade_cooldown: "Granaten Abklingzeit",
+          Melee_cooldown: "Nahkampf Abklingzeit",
           tier_progress: "{{progress}} für {{tier}}",
           About: "Über",
-          'Support DIM': "Unterstützen DIM",
+          'Support DIM': "DIM unterstützen",
```

Comment:
Capitalize both words to maintain current style. Same comment to translations below